src/connector/src/parser/mod.rs

```diff
@@ -132,6 +133,25 @@ impl SourceStreamChunkBuilder {
         builder.finish()
     }
 
+    fn finish_heartbeat(self) -> StreamChunk {
```

Comment:
Please help to leave some docs.

Reply:
+1 I think "heartbeat" is quite a hidden knowledge to others, and thus becomes fragile assumption. Should explain it more in the codebase.